azext_iot/_help.py

```diff
@@ -1031,22 +1209,58 @@
         az iot central app monitor-events --app-id {app_id} --output json
   """
 
-helps['iot central device-twin'] = """
+
+helps[
+    "iot central app validate-messages"
```

Comment:
In reviewing, I am not sure if it makes sense for `validate-messages` to be a separate command. It may be more apt and concise to have a `--validate-messages` / `--validate` / `--vm` switch against `iot central app monitor-events` as its practically the same flow. Why does this need to be a separate command?

Reply:
long story short: because this is just the first iteration on this command. The longer term goal of this command warrants it to have its own namespace